src/utils/index.ts

```diff
@@ -12,7 +12,8 @@ export const copyToClipboard = async (text: string): Promise<void> => {
 }
 
 export const formatAddress = (address: string, length = address.length / 2): string => {
-  return `${address.slice(0, length / 2)}...${address.slice(-length / 2)}`
+  const formatedAddress = `${address.slice(0, length / 2)}...${address.slice(-length / 2)}`
+  return formatedAddress.indexOf('0x') === 0 ? formatedAddress.substr(0, 2) + formatedAddress.substr(2).toUpperCase() : formatedAddress
```

Comment:
Don't understand why do we need to remove first 2 symbols?

Reply:
According to design we should uppercase only part after '0x'.

Reply:
Please discard your changes here according to address cryptography

Reply:
I reverted mentioned changes. Thank you for the comment!